src/components/earn/PoolCardTri/index.tsx

```diff
@@ -93,18 +116,18 @@ const DefaultPoolCardtri = ({
 
   const totalStakedInUSDFriendly = addCommasToNumber(totalStakedInUSD.toString())
 
+  const currenciesQty = currencies.length
+  const farmName = friendlyFarmName ?? currencies.map(({ symbol }) => symbol).join('-')
+
   function onCardClick() {
     setShowMore(!showMore)
   }
 
   function handleClaimClick(event: React.MouseEvent) {
-    enableModal()
+    enableClaimModal()
```

Comment:
Should `event.stopPropagation()` be invoked first?  Might cause a race condition

Reply:
Changed!